src/components/Tool/FixSendNewsletter/Contact.tsx

```diff
@@ -197,109 +197,101 @@ const Contact = ({
         className={classes.minimalPadding}
         sx={{ backgroundColor: theme.palette.cruGrayLight.main }}
       >
-        {contact.primaryPerson && (
-          <Grid container alignItems="center">
-            <Grid item xs={6} sm={5} md={4}>
-              <Box
-                display="flex"
-                alignItems="center"
-                style={{ height: '100%' }}
-              >
-                {contact.primaryPerson.firstName && matches && (
-                  <Avatar
-                    src={contact?.avatar}
-                    style={{
-                      width: theme.spacing(4),
-                      height: theme.spacing(4),
-                    }}
-                  />
-                )}
-                <Box display="flex" flexDirection="column" ml={2}>
-                  <Typography variant="subtitle1">
-                    {`${contact.primaryPerson.firstName} ${contact.primaryPerson.lastName}`}
-                  </Typography>
-                  <Link
-                    underline="hover"
-                    href={`mailto:${contact.primaryPerson.primaryEmailAddress?.email}`}
-                  >
-                    <Typography variant="body2">
-                      {contact.primaryPerson.primaryEmailAddress?.email || ''}
-                    </Typography>
-                  </Link>
-                  {contact.primaryPerson.optoutEnewsletter && (
-                    <Typography variant="body2">
-                      {t('opted out of newsletter')}
+        <Grid container alignItems="center">
+          <Grid item xs={6} sm={5} md={4}>
+            <Box display="flex" alignItems="center" style={{ height: '100%' }}>
+              {contact.primaryPerson?.firstName && matches && (
+                <Avatar
+                  src={contact?.avatar}
+                  style={{
+                    width: theme.spacing(4),
+                    height: theme.spacing(4),
+                  }}
+                />
+              )}
+              <Box display="flex" flexDirection="column" ml={2}>
+                {contact.primaryPerson?.firstName &&
+                  contact.primaryPerson?.lastName && (
+                    <Typography variant="subtitle1">
+                      {`${contact.primaryPerson?.firstName} ${contact.primaryPerson?.lastName}`}
```

Comment:
We already checked that `primaryPerson`, `firstName`, and `lastName` are truthy.

```suggestion
                      {`${contact.primaryPerson.firstName} ${contact.primaryPerson.lastName}`}
```